### DISCUSSION TITLE
(from) Google Chrome data import

### DISCUSSION THREAD

**Discussion by NukeThief** [Eloston/ungoogled-chromium#2067] [Ideas]
I'm a Google Chrome user, and i want to start using ungoogled-chromium for multiple reasons, but there is one thing holding me back, there is no data import from Chrome so please if possible add such feature.

**Comment by networkException** [comment]
You should be able to just copy your chrome profile into the directory that ungoogled-chromium uses. What directory that is depends on your platform

**Reply by NukeThief** [reply]
you know where it is? cuz i dont

**Reply by NukeThief** [reply]
found it
AppData\Local

**Reply by NukeThief** [reply]
Didn't work properly when i dont replace files it dosent fully import but when i replace ug-c breaks.

**Reply by networkException** [reply]
Removing the `Default` directory in chromium and replacing it with the one from chrome worked fine for me

![image](https://user-images.githubusercontent.com/42888162/187551121-f7640351-a739-4b9d-befc-75426c593e98.png)

**Reply by NukeThief** [reply]
This time it was even better but my passwords are gone but i just exported and importex them Thanks!

**Reply by NukeThief** [reply]
Now i can enjoy Chrome (but better(for many reasons))!

**Comment by NukeThief** [comment]
ok

**Comment by NukeThief** [comment]
Didn't work properly when i dont replace files it dosent fully import but when i replace ug-c breaks.

**Comment by NukeThief** [comment]
i still wnat help!
